README.md

```diff
@@ -1,33 +1,38 @@
 # another-go-installer
 
-Script to install `Golang` and create a basic [workspace](https://golang.org/doc/code.html). By default the script takes the lates version in the download [page](https://golang.org/dl/)
+Script to install `Golang` and create a basic [workspace](https://golang.org/doc/code.html#Workspaces). By default the script takes the latest version in the download [page](https://golang.org/dl/).
 
 ## Features
 
 ✔ Detects automatically the Operative System (`mac`, `linux`)
 
 ✔ Detects automatically the architecture of your OS (`64`, `32` bits)
 
-❌ Prompt for a github user (to create projects directory inside the workspace)
+❌ Prompts for a github user (to create projects directory inside the workspace)
 
-❌ Install a specific version
+❌ Installs a specific version
```

Comment:
And I think another thing that this should have is another feature like 👇 (another feature not yet implemented)

❌ Validate checksum of the downloaded file

Can you add this on this PR 😅